modules/light-clients/08-wasm/internal/ibcwasm/expected_interfaces.go

```diff
@@ -0,0 +1,109 @@
+package ibcwasm
+
+import (
+	wasmvm "github.com/CosmWasm/wasmvm"
+	wasmvmtypes "github.com/CosmWasm/wasmvm/types"
+)
+
+var _ WasmEngine = (*wasmvm.VM)(nil)
+
+type WasmEngine interface {
+	// Create will compile the wasm code, and store the resulting pre-compile
```

Comment:
this may be fixed from another backport, but just doing to raise this here as I did on the other backport PR.

```suggestion
	// StoreCode will compile the wasm code, and store the resulting pre-compile
```